docs/development/local_setup.md

```diff
@@ -69,31 +105,76 @@ $ git clone git@github.com:gardener/gardener.git
+Starting cluster components...
+Kubectl is now configured to use the cluster.
+Loading cached images from config file.
+
+$ make dev-setup
+clusterrolebinding "kube-dns-rule" created
+namespace "garden" created
+secret "default-domain" created
+secret "internal-domain-example-com" created
+deployment "etcd" created
+service "etcd" created
+```
 
-Please take a look at the [example manifests folder](../../example) to see which resource objects you need to install into your Garden cluster.
+Next, you need to run the API server and the Gardener controller manager using rules in the `Makefile`.
 
-For your convenience, we have created a rule in the `Makefile` which will automatically start the Gardener controller manager with development settings:
+```bash
+$ make dev-apiserver
+[restful] 2018/02/01 15:39:43 log.go:33: [restful/swagger] listing is available at https:///swaggerapi
+[restful] 2018/02/01 15:39:43 log.go:33: [restful/swagger] https:///swaggerui/ is mapped to folder /swagger-ui/
+I0201 15:39:43.750573   84958 serve.go:89] Serving securely on [::]:8443
+```
+
+In another terminal, you must launch the Gardener controller manager:
 
 ```bash
 $ make dev
-time="2017-12-05T15:58:48+01:00" level=info msg="Starting Gardener controller manager..."
-[...]
-time="2017-12-05T15:58:55+01:00" level=info msg="Watching only namespace 'johndoe' for Shoot resources..."
-time="2017-12-05T15:58:55+01:00" level=info msg="Gardener controller manager (version 0.28.0) initialized successfully."
+time="2018-01-29T10:18:37+02:00" level=info msg="Starting Gardener controller manager..."
+time="2018-01-29T10:18:38+02:00" level=info msg="Gardener controller manager HTTP server started (serving on 0.0.0.0:2718)"
+time="2018-01-29T10:18:38+02:00" level=info msg="Found default domain secret default-domain for domain 192.168.99.100.nip.io."
+time="2018-01-29T10:18:38+02:00" level=info msg="Found internal domain secret internal-domain-nip-io for domain 192.168.99.100.nip.io."
+time="2018-01-29T10:18:38+02:00" level=info msg="Successfully bootstrapped the Gardener cluster."
+time="2018-01-29T10:18:38+02:00" level=info msg="Gardener controller manager (version 0.1.0) initialized."
+time="2018-01-29T10:18:38+02:00" level=info msg="Watching all namespaces for Shoot resources..."
+time="2018-01-29T10:18:38+02:00" level=info msg="Shoot controller initialized."
+time="2018-01-29T10:18:38+02:00" level=info msg="Seed controller initialized."
 ```
 
 :information_source: Your username is inferred from the user you are logged in with on your machine. The version is incremented based on the content of the `VERSION` file. The version is important for the Gardener in order to identify which Gardener version has last operated a Shoot cluster.
 
-The Gardener should now be ready to operate on Shoot resources associated to your namespace.
+The Gardener should now be ready to operate on Shoot resources. You can use
+
+```bash
+$ kubectl get shoots
+No resources found.
+```
```

Comment:
Can we mention here that the user can assume that the setup has worked once `No resources found` is returned?

Reply:
I've added a note in the section bellow